learning/prompts/documentation-lookup/03_basic_configuring_pipelines.md

```diff
@@ -0,0 +1,29 @@
+Prompt:
+How do I configure pipeline in Apache Beam?
```

Comment:
Is it intentional that the prompt doesn't include the word "options" after pipeline? Just want to flag this in case it wasn't on purpose.

Reply:
Thank you. I was thinking about a user asking how to configure a pipeline in a broader sence. He might not know that Beam uses term options. It might be parameters, settings so basically this is a question about configuration.